datasets/us/de/med_exclusions/crawler.py

```diff
@@ -0,0 +1,89 @@
+from typing import Dict
+from rigour.mime.types import PDF
+from datetime import datetime
+import re
+
+from zavod import Context, helpers as h
+from zavod.shed.zyte_api import fetch_resource
+
+DATE_PATTERN = r"\b\d{1,2}/\d{1,2}/\d{4}\b"
+REGEX_AKA = re.compile(r"\(?a\.?k\.?a\b\.?|\)", re.IGNORECASE)
+REGEX_DBA = re.compile(r"d.b.a.", re.IGNORECASE)
+
+
+def crawl_item(row: Dict[str, str], context: Context):
+
+    raw_name = row.pop("sanctioned_provider_name")
+
+    entity = context.make("LegalEntity")
+    entity.id = context.make_id(raw_name, row.get("npi"))
+
+    names = REGEX_AKA.split(raw_name)
+    name, alias = names[0], names[1:]
+
+    if "d.b.a." in name:
+        names = REGEX_DBA.split(name)
+        name, dba = names[0], names[1]
+
+        dba_company = context.make("Company")
+        dba_company.id = context.make_id(dba)
+        dba_company.add("name", dba)
+        dba_company.add("country", "us")
+        dba_company.add("topics", "debarment")
+        link = context.make("UnknownLink")
+        link.id = context.make_id(entity.id, dba_company.id)
+        link.add("object", dba_company)
+        link.add("subject", entity)
+        link.add("role", "d/b/a")
+        context.emit(dba_company)
+        context.emit(link)
+
+    entity.add("name", name)
+    entity.add("alias", alias)
+    entity.add("country", "us")
+
+    if row.get("npi") != "N/A":
+        npis = h.multi_split(row.pop("npi"), [";", "&", " and ", ","])
+        entity.add("npiCode", npis)
+    else:
+        row.pop("npi")
+
+    if row.get("license") != "N/A":
+        entity.add("description", "License Number: " + row.pop("license"))
+    else:
+        row.pop("license")
```

Comment:
We've been discussing internally that we don't want to build up `description` to be a bag of key-value pairs - that's made a mess of some of the upstream data and we don't want to repeat that. So let's put `license` into `idNumber` for now - that prop is fine for identifiers that don't fit into a more specific type like `registrationNumber` or `npiCode`